src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { Auth0Provider } from "@auth0/auth0-react";
+
+ReactDOM.render(
```

Comment:
https://github.com/reactwg/react-18/discussions/5

After React18, the `ReactDOM.render` method has been deprecated, use `createRoot` instead